lib/rest/rest.go

```diff
@@ -292,11 +292,39 @@ func (api *Client) Call(opts *Opts) (resp *http.Response, err error) {
 // contentName - the name of the parameter for the file
 //
 // NB This doesn't allow setting the content type of the attachment
-func MultipartUpload(in io.Reader, params url.Values, contentName, fileName string) (io.ReadCloser, string, error) {
+func MultipartUpload(in io.Reader, params url.Values, contentName, fileName string) (io.ReadCloser, string, *int64, error) {
```

Comment:
Why is the return an `*int64`?  I would have thought an `int64` would be fine? That saves problems with `nil` pointers etc.